### PR TITLE
Add facebook Pixel as plugin

### PR DESCRIPTION
**Pull request by gris**
## What's the purpose of this pull request?
Adds the facebook pixel integration as a plugin. It is very similar to https://github.com/vtex-apps/facebook-pixel. I'm still not sending the currency to FB servers (the GTM plugin also does not as well).

## How it works? 
<em>Tell us the role of the new feature, or component, in its context.</em>

## How to test it?
You can use this chrome extension to verify that the events are being sent https://chrome.google.com/webstore/detail/facebook-pixel-helper/fdgfkebogiimcoedlicjlajpkdmockpc

## References
<em>Spread the knowledge: is there any content you used to create this PR that is worth sharing?</em>  

<em>Extra tip: adding references to related issues or mentioning people important to this PR may be good for the documentation and reviewing process</em>

### CODE REVIEW

File: packages/gatsby-plugin-pixel-facebook/src/pixel/index.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  /* eslint-disable */
+  useMemo(() => setupFBPixel(pixelId), [])
+
+interface Props {
+  pixelId: string
+  timeout?: number
+}
+
+const Provider: FC<Props> = ({ children, pixelId, timeout = 2000 }) => {
```
Comment (gris) [inline packages/gatsby-plugin-pixel-facebook/src/pixel/index.tsx:46]: Is this timeout too tight?

**Review by tlgimenes — CHANGES_REQUESTED** [review]
please make a prod build

File: packages/gatsby-plugin-pixel-facebook/src/pixel/index.tsx (hunk 1 above)
Comment (tlgimenes) [reply packages/gatsby-plugin-pixel-facebook/src/pixel/index.tsx:46]: I think so. We usually use 5.5e3 seconds

**Comment by gris** [comment]
@tlgimenes I've tried doing a `yarn build` both on the root folder and on the `gatsby-theme-store` file, but the generated files are still there. How can I fix this?

**Comment by tlgimenes** [comment]
You need to `yarn build` on the store, so the queries are build with production

**Review by klzns — APPROVED** [review]

File: packages/gatsby-theme-store/src/sdk/buyButton/useBuyButton.ts [hunk 2]
```diff
@@ -83,7 +83,7 @@ export const useBuyButton = ({
       sendPixelEvent({
         type: 'vtex:addToCart',
         data: {
-          items,
+          items: [{ ...orderFormItem, price: seller.commercialOffer.price }],
```
Comment (tlgimenes) [inline packages/gatsby-theme-store/src/sdk/buyButton/useBuyButton.ts:86]: isn't it better to add this to orderFormItem in line 75 ?

**Review by tlgimenes — APPROVED** [review]
LGTM, just one small change

**Pull request merged**